src/dir.targets

```diff
@@ -1,3 +1,9 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <!-- metadata used for Authenticode signing in official VSO builds -->
+  <ItemGroup>
```

Comment:
I would prefer to always include it.  A little extra metadata that goes unused for dev builds shouldn't have any adverse affects.  Also, if for some reason this does become a problem we should have an early read instead of the VSO builds suddenly breaking.